source/expressions.tex

```diff
@@ -1270,7 +1270,7 @@
 \pnum
 An \grammarterm{id-expression}
 that denotes an immediate function\iref{dcl.constexpr}
-shall appear as a subexpression of an immediate invocation or
+shall only appear as a subexpression of an immediate invocation or
```

Comment:
I like Walter's suggestion of adding bullets and moving the only a little farther towards the end.